gbpservice/nfp/orchestrator/modules/service_orchestrator.py

```diff
@@ -251,17 +262,21 @@ def network_function_device_notification(self, context, notification_data):
                 elif operation == 'delete':
                     event_id = self.rpc_event_mapping[resource][1]
                 elif operation == 'pt_add':
+                    serialize = True
```

Comment:
- Isn't the issue with simultaneous member addition through heat template? If so, why are policy target operations serialized?
- Don't understand the consumer add operation also. Can we discuss?